android/src/main/java/com/thebluealliance/androidclient/activities/BaseActivity.java

```diff
@@ -31,6 +38,25 @@
 public abstract class BaseActivity extends NavigationDrawerActivity
         implements NfcAdapter.CreateNdefMessageCallback, GoogleApiClient.ConnectionCallbacks, GoogleApiClient.OnConnectionFailedListener {
 
+    public enum WarningMessageType {
```

Comment:
Wait, actually use an [IntDef](http://developer.android.com/reference/android/support/annotation/IntDef.html) for this, in its own class. Use the value for the priority